packages/extension/src/components/organisms/DisplayTransactionDetails/DisplayTransactionDetails.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+
+import { Transaction } from 'xrpl';
+
+import { DataCard, RawTransaction, XRPLTransaction } from '../../molecules';
+import { Fee } from '../../organisms';
+import { LoadingOverlay } from '../../templates';
+
+interface Props {
+  txParam: Transaction | null;
+  estimatedFees: string;
+  errorFees?: string;
+}
+
+export const DisplayTransactionDetails: React.FC<Props> = ({
```

Comment:
Why name it `DisplayTransactionDetails` and not `TransactionDetails`

Reply:
```suggestion
export const TransactionDetails: FC<Props> = ({
```
You can import `FC` from React directly.